test/e2e/set/scyllacluster/verify.go

```diff
@@ -113,4 +113,10 @@ func verifyScyllaCluster(ctx context.Context, kubeClient kubernetes.Interface, s
 	o.Expect(err).NotTo(o.HaveOccurred())
 	o.Expect(status.DownHosts()).To(o.BeEmpty())
 	o.Expect(status.LiveHosts()).To(o.HaveLen(memberCount))
+
+	if di != nil {
+		read, err := di.Read()
+		o.Expect(err).NotTo(o.HaveOccurred())
```

Comment:
By("verifying ..

Reply:
nit: verify logs line should come after the read so it's (writing -> reading -> verifying)